frontend/src/component/project/Project/ProjectSettings/ChangeRequestConfiguration/ChangeRequestConfiguration.tsx

```diff
@@ -1,313 +1,67 @@
-
-                    return (
-                        <ConditionallyRender
-                            condition={original.changeRequestEnabled}
-                            show={
-                                <StyledBox data-loading>
-                                    <GeneralSelect
-                                        options={approvalOptions}
-                                        value={original.requiredApprovals || 1}
-                                        onChange={approvals => {
-                                            onRequiredApprovalsChange(
-                                                original,
-                                                approvals
-                                            );
-                                        }}
-                                        disabled={
-                                            !hasAccess(
-                                                UPDATE_PROJECT,
-                                                projectId
-                                            )
-                                        }
-                                        IconComponent={
-                                            KeyboardArrowDownOutlined
-                                        }
-                                        fullWidth
-                                    />
-                                </StyledBox>
-                            }
-                        />
-                    );
-                },
-                width: 100,
-                disableGlobalFilter: true,
-                disableSortBy: true,
-            },
-            {
-                Header: 'Status',
-                accessor: 'changeRequestEnabled',
-                id: 'changeRequestEnabled',
-                align: 'center',
-
-                Cell: ({ value, row: { original } }: any) => (
-                    <StyledBox data-loading>
-                        <PermissionSwitch
-                            checked={value}
-                            projectId={projectId}
-                            permission={UPDATE_PROJECT}
-                            inputProps={{ 'aria-label': original.environment }}
-                            onClick={onRowChange(
-                                original.environment,
-                                original.changeRequestEnabled,
-                                original.requiredApprovals
-                            )}
-                        />
-                    </StyledBox>
-                ),
-                width: 100,
-                disableGlobalFilter: true,
-                disableSortBy: true,
-            },
-        ],
-        []
+    const projectName = useProjectNameOrId(projectId);
+    const { hasAccess } = useContext(AccessContext);
+    const { isOss, uiConfig } = useUiConfig();
+    const isPro = !(
+        Boolean(uiConfig.versionInfo?.current.oss) ||
```

Comment:
uiConfig has `isOSS`

Reply:
Would be nice to have this in a reusable hook or util: `const { isOss, isPro, isEnteprise } = usePlan()` or something. There's also `useInstanceStatus`.

Reply:
Here's something that may help: https://github.com/Unleash/unleash/pull/2600